packages/java/parser-jvm-utils/src/main/java/com/vaadin/fusion/parser/utils/ConfigList.java

```diff
@@ -0,0 +1,50 @@
+package com.vaadin.fusion.parser.utils;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Objects;
+import java.util.Set;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public interface ConfigList<Item> {
```

Comment:
I think this class needs some java doc or maybe better method names. 
why getDisable() returns a Collection of Item? 
the same for getUse()

Reply:
Hmm, well, `getDisable` is a getter that returns `disable` property which is a maven-level configuration part:
```xml
<plugin>
  <configuration>
    <plugins>
      <disable>
        <plugin>com.vaadin.fusion.parser.plugins.backbone.BackbonePlugin</plugin>
      </disable>
      <use>
        <plugin>com.vaadin.fusion.parser.plugins.nonnull.NonnullPlugin</plugin>
      </use>
  </configuration>
</plugin
```
I thought I cannot change the property name since it should be read by maven. But probably I could use something like `@JsonProperty` of Jackson? Or I should just use another name for the getter?

Reply:
No need to use `@JsonProperty`, could you help to add this explanation as JavaDoc to what the `disable` and `use` do? It would help to understand how things work in the future.

Reply:
Done!